### PR TITLE
This should quieten a couple of warnings

### PR DESCRIPTION
**Pull request by mischat**
This should address this issue https://github.com/bosonprotocol/boson-protocol-contracts/issues/619

### CODE REVIEW

File: contracts/protocol/facets/ExchangeHandlerFacet.sol [hunk 1]
```diff
@@ -24,7 +24,8 @@ import { IERC20 } from "../../interfaces/IERC20.sol";
 contract ExchangeHandlerFacet is IBosonExchangeHandler, BuyerBase, DisputeBase {
     using Address for address;
 
-    uint256 private immutable EXCHANGE_ID_2_2_0;
+    uint256 private immutable EXCHANGE_ID_2_2_0; // solhint-disable-next-line
```
Comment (zajck) [inline contracts/protocol/facets/ExchangeHandlerFacet.sol:27]:
```suggestion
    uint256 private immutable EXCHANGE_ID_2_2_0; // solhint-disable-line
```

**Review by zajck — CHANGES_REQUESTED** [review]

**Review by anajuliabit — APPROVED** [review]

**Review by anajuliabit — APPROVED** [review]

**Review by anajuliabit — DISMISSED** [review]

File: contracts/protocol/facets/ExchangeHandlerFacet.sol (hunk 1 above)
Comment (mischat) [reply contracts/protocol/facets/ExchangeHandlerFacet.sol]: This should be fixed.

**Comment by mischat** [comment]
indeed, i will look at it, and make sure it is OK before pushing it up again... @zajck

**Comment by mischat** [comment]
finally, this should work i guess ... Let's wait for the tests etc

**Review by zajck — APPROVED** [review]

**Review by anajuliabit — APPROVED** [review]

**Pull request merged**